### ISSUE TITLE
still happening: Error: Please pass numbers as strings or BigNumber objects to avoid precision errors.

### ISSUE THREAD

**Issue by syzygyzer** [urbit/bridge#337]
This sort of error was at least partially handed by #321 , but I have encountered it again.

I previously encountered it when attempting to reticket: https://github.com/urbit/bridge/issues/306#issuecomment-547003530

After #321, I was successfully able to reticket from the Admin menu on the Bridge Web UI, so that has been addressed.

I have just encountered it again, though, when attempting to Reset Networking Keys with Trigger a New Continuity Era checked from the Admin/Edit Permissions/Networking Keys tab in the Bridge Web UI. This error occurs after the transaction has been signed and the send transaction button was pressed.

```
Error: Please pass numbers as strings or BigNumber objects to avoid precision errors.
    at fromWei (https://bridge.urbit.org/static/js/2.3eaa8aad.chunk.js:1:69433)
    at jn (https://bridge.urbit.org/static/js/main.3ca5b517.chunk.js:1:123182)
    at Na (https://bridge.urbit.org/static/js/2.3eaa8aad.chunk.js:1:1561602)
    at Go (https://bridge.urbit.org/static/js/2.3eaa8aad.chunk.js:1:1604694)
    at Fo (https://bridge.urbit.org/static/js/2.3eaa8aad.chunk.js:1:1590417)
    at qo (https://bridge.urbit.org/static/js/2.3eaa8aad.chunk.js:1:1587437)
    at Lo (https://bridge.urbit.org/static/js/2.3eaa8aad.chunk.js:1:1585997)
    at https://bridge.urbit.org/static/js/2.3eaa8aad.chunk.js:1:1547898
    at t.unstable_runWithPriority (https://bridge.urbit.org/static/js/2.3eaa8aad.chunk.js:1:1622341)
    at pi (https://bridge.urbit.org/static/js/2.3eaa8aad.chunk.js:1:1547627)
```

This error didn't seem to be dependent on the browser or OS when I last encountered it, but just in case, this is running in Brave on NixOS.

**Comment by syzygyzer** [comment]
I should also have noted, in case it matters, that when I reticketed I needed to transfer some of my own ETH to the listed address (about $2.50 worth, I think) for it to work. I don't know enough about Bridge yet to know whether that is somehow related, or expected behavior and unrelated to this error.

**Comment by Fang-** [comment]
>I needed to transfer some of my own ETH to the listed address (about $2.50 worth, I think) for it to work.

That is related in that this error likely comes from trying to render the desired amount as ETH, as in the screenshot below.

(We had deployed an un-minimized version of the codebase for better stack traces, but earlier today I overwrote that with a minimized version... Your stack trace would likely point in this direction.)

<img width="486" alt="image" src="https://user-images.githubusercontent.com/3829764/68413253-2d95d780-018e-11ea-9241-33c01b153f5d.png">

Can confirm it doesn't seem browser-dependent, but rather is influenced by cosmic background radiation.

**Comment by syzygyzer** [comment]
I should note that I do not have the "not enough ETH" message when I attempt to send transaction (but I did get that message when I successfully reticketed after #321). I can send some ETH to my ownership address though and see if that changes any behavior. Though to be clear - should I send to the Ownership Address or the Management Proxy address, or some other address? Right now, both addresses are empty for my ship.

**Comment by Fang-** [comment]
Wait, you're able to _consistently_ reproduce this error? If that's the case, and it's not too big a deal, please don't touch anything right now, I'd love to see if I can figure out what's triggering this.

When, exactly, does this happen, if not when/directly after you click "send transaction"?

**Comment by syzygyzer** [comment]
Yeah, the error is consistent. I've tried it a half dozen more times and it comes up every time.

It happens directly after I click Send Transaction. A new page loads that says Bridge Error etc, followed by the error message.

I'll stop touching my ship for now. Let me know if there's anything I can grab to help with diagnostics.

**Comment by syzygyzer** [comment]
@Fang- and I talked thru email and he tweaked something and this error went away for me. I was left with the prompt begging for ETH, and upon paying the ransom my personal breach completed successfully.

**Comment by Fang-** [comment]
Closing as duplicate of https://github.com/urbit/bridge/issues/410#issuecomment-597855487.
